File: ArtGalleryAPI/ArtGalleryAPI/Models/Dto/TopSellingProductDto.cs
namespace ArtGalleryAPI.Models.Dto
{
    public class TopSellingProductDto
    {
        public Guid ProductId { get; set; } // ProductId is of type Guid
        public string Name { get; set; } // Product Name
        public decimal TotalSales { get; set; } // Total sales, assuming it's a decimal
    }

}

File: ArtGalleryUI/src/app/features/statistics/models/TopSellingProductDto.ts
export class TopSellingProductDto {
    productId: string;
    name: string;
    totalSales: number = 0; // Initialize with a default value

    constructor(productId: string, name: string, totalSales: number = 0) {
        this.productId = productId;
        this.name = name;
        this.totalSales = totalSales; // Can also be set in the constructor
    }
}
